src/ponyc/main.c

```diff
@@ -118,6 +120,9 @@ static void usage()
     "  --stats         Print some compiler stats.\n"
     "\n"
     "Debugging options:\n"
+    "  --verbose       Verbosity level: 0: quiet, 1: default, \n"
+    "                                   2: compile info, 3: tool info\n"
+    "                                   4: everything\n"
```

Comment:
It seems like this should do a better job of explaining what is a part of each verbosity level.  From this description alone, I don't think I would get it.

Maybe you can copy the style of some of the below options, which list each allowed value.  Something like:
```
--verbose, -v
  =0            Print only [...].
  =1            Print only [...] and [...].
  [...]
```